docs/apache-airflow-providers-google/operators/cloud/bigquery.rst

```diff
@@ -42,7 +42,7 @@ Create dataset
 To create an empty dataset in a BigQuery database you can use
 :class:`~airflow.providers.google.cloud.operators.bigquery.BigQueryCreateEmptyDatasetOperator`.
 
-.. exampleinclude:: /../../airflow/providers/google/cloud/example_dags/example_bigquery_operations.py
+.. exampleinclude:: /../../tests/system/providers/google/bigquery/example_bigquery_dataset.py
```

Comment:
We have to check if the example include directive is correctly used. I think you might need to update it.
https://github.com/apache/airflow/blob/main/docs/exts/docs_build/lint_checks.py#L232-L247

Reply:
I see, we will need to update the path in the linked lint check to math two possible paths (example_dags and tests/system),

Reply:
This lint check actually doesn't work at all :) For example:
```
f"${DOCS_DIR}/**/*.rst"
```
``$`` looks like a typo - after removing it and running the check I'm getting the errors (for files I didn't modify - this check was silently ignoring them). I will fix the check.

Reply:
Done